### PR TITLE
Added Shadow Role section for each events-team and Activities to Project Management events team

### PR DESCRIPTION
**Pull request by hpandeycodeit**
- Added `Shadow Role` section to each of the events-team Best Practices doc. 
- Added Activites to the Project Management Best Practices.

### CODE REVIEW

File: events/events-team/marketing/README.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ As Marketing Lead, you are responsible for the overall marketing communications
 - Determine signage needs and copy, coordinate with CNCF on explicit needs  
 - Create/update deck templates for curated talks and/or other purposes
 
+## Shadow to the Marketing Lead Role
+This role can include a shadow role as well. The Marketing Lead can delegate/assign the tasks to the shadow.The expectation from a shadow is to lead one of the events-team in the next summit. The shadow to the Marketing Lead is responsible for handling all Registration lead related activities in the absence of the Lead.
```
Comment (tpepper) [inline events/events-team/marketing/README.md]: Copy/paste error: should be "...responsible for handling all Marketing lead related activities..."

File: events/events-team/projectmanagement/README.md [hunk 2]
```diff
@@ -18,7 +21,14 @@ Good communication
 Impartial and objective
 
 # Activities
-Ensure the overall event is a great experience for all contributors Recruit a core team (see best-practices.md) and help recruit other volunteers as necessary
-Create an event strategy with the Content Lead(s) and consider having a social event (decide if social needs it's own lead) Provide venue ideas and suggestions to CNCF; do a walk thru if possible
-Organize and lead n cadence update meetings
-Direct liaison with CNCF Event Manager
\ No newline at end of file
+* Ensure the overall event is a great experience for all contributors Recruit a core team (see best-practices.md) and help recruit other volunteers as necessary.
```
Comment (tpepper) [inline events/events-team/projectmanagement/README.md]: Suggest adding a '.' in the middle of "...contributors Recruit..."

File: events/events-team/marketing/README.md (hunk 1 above)
Comment (hpandeycodeit) [reply events/events-team/marketing/README.md]: good catch :) fixed it.

File: events/events-team/projectmanagement/README.md (hunk 2 above)
Comment (hpandeycodeit) [reply events/events-team/projectmanagement/README.md]: this too. :)

**Comment by tpepper** [comment]
Looks good to me.  Would prefer a squash of the commits.

**Comment by hpandeycodeit** [comment]
done. 

> Looks good to me. Would prefer a squash of the commits.

**Comment by cblecker** [comment]
I don't know why the bot is suggesting me lol
/unassign
/assign @mrbobbytables @parispittman @jberkus

File: events/events-team/projectmanagement/README.md [hunk 3]
```diff
@@ -7,6 +7,9 @@ The Project Manager(s) is responsible for Triaging the project board, related is
 Also, keep track of the timelines of these tasks and outlined activited. 
 The Project Manager role will ideally have a shadow for succession planning. Those who have been a part of a previous event get first dibs on lead roles to maintain consistency.
 
+## Shadow to the Project Manager Role
+This role can include a shadow role as well. The PM can delegate/assign the tasks to the shadow.The expectation from a shadow is to lead one of the events-team in the next summit. The shadow is responsible for handling all Project Management related activities in the absence of the PM.
```
Comment (mrbobbytables) [inline events/events-team/projectmanagement/README.md]:
```suggestion
This role can include one or more shadows. The PM can delegate/assign tasks to the shadows. The expectation from a shadow is to lead one of the events-team roles in an upcoming summit. The shadows are responsible for handling all Project Management related activities in the absence of the PM.
```

Similar to the above, just some suggestions to make it clear that one or more shadows are possible 👍

File: events/events-team/registration/README.md [hunk 4]
```diff
@@ -43,6 +43,9 @@ A successful Registration lead should have the following qualities:
   comfortable with queries and syntax is incredibly helpful, both for for the
   registration approval process and later report generation.
 
+## Shadow to the Registration Lead Role
+This role can include a shadow role as well. The Registration Lead can delegate/assign the tasks to the shadow.The expectation from a shadow is to lead one of the events-team in the next summit. The shadow to the Registration Lead is responsible for handling all Registration lead related activities in the absence of the Lead.
```
Comment (mrbobbytables) [inline events/events-team/registration/README.md]:
```suggestion
This role can include one or more shadows. The Registration Lead can delegate/assign the tasks to the shadows. The expectation from a shadow is to lead one of the events-team roles in an upcoming summit. The shadows to the Registration Lead are responsible for handling all Registration lead related activities in the absence of the Lead. 
```

More of the same 👍

**Comment by parispittman** [comment]
thanks for this @hpandeycodeit!! happy to approve after @mrbobbytables comments are addressed

**Comment by hpandeycodeit** [comment]
Thanks @mrbobbytables @parispittman! I have made the requested changes :)

**Comment by mrbobbytables** [comment]
Thanks @hpandeycodeit :)
/lgtm

**Comment by parispittman** [comment]
/approve

thank you so much!!

**Pull request merged**